pam/vplugin/vplugin.go

```diff
@@ -40,6 +40,7 @@ func RegisterGobs() {
 	gob.RegisterName("configs.TraceConfig", configs.TraceConfig{})
 
 	gob.Register(map[string]interface{}{})
+	gob.Register(new(interface{}))
```

Comment:
not possible to register with `pam.SettlementType`? 
`gob.Register("pam.SettlementType", pam.SettlementType{})` Then that would be used in the `GetSettlementType`-function as well

Reply:
This is for the params sent to the plugin call, which in this case is practically nil as there are no input parameters to the function.... Don't think I need to register the pam.SettlementType as it is essentially just a string and not a struct, but that return type will ofc be used in the function.

Reply:
Just maybe add a comment why its registered, since it looks a bit strange